src/main/java/net/snowflake/client/core/ExecTimeTelemetryData.java

```diff
@@ -5,24 +5,19 @@
 
 import com.google.common.base.Strings;
 import net.minidev.json.JSONObject;
-import net.snowflake.client.jdbc.SnowflakeUtil;
 import net.snowflake.client.jdbc.telemetryOOB.TelemetryService;
+import net.snowflake.client.util.TimeMeasurement;
 
+@SnowflakeJdbcInternalApi
```

Comment:
I am not sure we should make the whole class internal when previously it was public

Reply:
I think that we are safe - the class does not make any sense outside of the driver. We do not want the user to provide any arbitrary metrics and send them to our telemetry endpoint.

Reply:
After the chat, agreed to remove it from the already public class.